start.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Set the environment variables.
+source ./set_env_variables.sh
+
+build=false
+
+# If the `-b` flag is passed, set build to true.
+while getopts b: flag; do
+    case ${flag} in
+        b) build=true;;
+    esac
+done
+
+check_status() {
+    status_code=$(curl --write-out %{http_code} --silent --output /dev/null localhost:${FLASK_RUN_PORT}/graphiql)
+    if [[ ${iterator} -lt 35 && ${status_code} -eq 200 || ${status_code} -eq 302 || ${status_code} -eq 400 ]]
+    then
+        >&2 echo -e "${GREEN}GraphiQL is Up at localhost:${FLASK_RUN_PORT}/graphiql${NC}"
```

Comment:
This should open the page. Again it should be the right port and not try to open graphiql.